P5/Source/Guidelines/en/CMC-ComputerMediatedCommunication.xml

```diff
@@ -0,0 +1,1302 @@
+        example, the second is a reply to the first, and the third is a reply to the second.
+        <egXML xml:id="e9" xmlns="http://www.tei-c.org/ns/Examples" xml:lang="de" source="#BIB_scilog1">
+          <post type="comment" modality="written" generatedBy="human" xml:id="cmc_post10" who="#u7"
+            replyTo="#cmc_ex_p9" when-iso="2015-07-29T21:44">
+            <p>Es hat den Anschein, als wäre bei BER durchaus große Kompetenz am Bau, allerdings
+              nicht in Form von Handwerkern….</p>
+            <p>http://www.zeit.de/2015/29/imtech-flughafen-berlin-ber-verzoegerung/komplettansicht</p>
+          </post>
+          <post type="comment" modality="written" generatedBy="human" xml:id="cmc_post11" who="#u8"
+            replyTo="#cmc_ex_p10" when-iso="2015-07-30T19:11">
+            <p>Nein Nein, an den Handwerkern kann es rein strukturel nicht gelegen haben. Niemand
+              lässt seine Handwerker auf der Baustelle derart allein. Zudem gibt es höchstoffizielle
+              “Abnahmen” von Bauabschnitten/phasen. Welcher Mangel auch bestanden hatte, er hätte
+              Zeitnah auffallen müssen.</p>
+            <p>Uuups, für Imtek hab ich mal in einer Nachunternehmerfirma gearbeitet. Imtek is
+              offenbar ein universeler Bauträger, der alles baut.</p>
+          </post>
+          <post type="comment" modality="written" generatedBy="human" xml:id="cmc_post12" who="#u8"
+            replyTo="#cmc_ex_p11" when-iso="2015-07-30T19:26">
+            <p>Stahlkunstruktionen dacht ich mal, was die bauen — oder bauen lassen.</p>
+            <p>Das ist schon ein übles Ding. Die Ausschreibungenund Angebote sind unauffällig, aber
+              wenn Unregelmässigkeiten auftreten (im Bauverlauf) dann gibt es die saftigen
+              Rechnungen. Da steht dann der Bauherr da und fragt sich, wie er denn so schnell einen
+              fähigen Ersatz herbekommt. Und diese Frage erübrigt sich meist, weil der Markt der
+              Baufirmen das nicht hergibt — weil tendenziel 100 % Auslastung. (und noch schlimmer:
+              Absprachen) Was auch Folge des Marktdrucks gewesen war.</p>
+          </post>
+        </egXML>
+      </p>
+      <p>In the CMC genre of Wiki talk, users insert their contribution to a discussion by modifying
+        the wiki page of the discussion — the talk page. Since there is no technical reply action
+        available in wiki software, users apply textual indentation in the wiki code to indicate a
+        reply to a previous message, and a threaded structure is formed by a series of such
+        indentations. The attribute <att>indentLevel</att> records the level of indentation, that is
+        the nesting depth of the current post in such a thread-like structure (as defined by its
+        author and in relation to the standard level of non-indentation which should be encoded with
+        an <att>indentLevel</att> of <val>0</val>). It is used in wiki talk corpora but may also be
```

Comment:
Is an attribute `@indentLevel` really necessary? Couldn't you just use `@replyTo`? Both convey the meaning that there one post is a reply to another post, only that in one case it is done via 'replyTo' and in another it is done by indentation.

Or is the question of sequence related (mentioned later in the chapter)? Perhaps that should be mentioned here?